test/auth-create-reset.spec.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const EmailTemplate = require('email-templates').EmailTemplate;
+
+const supportedLocales = [
```

Comment:
Why not have this file seperate as it's shared in them all 
Also maybe in json so it can be parsed by jq to get the list to download in the build process

Reply:
Not all files have the same `supportedLocales`, and it may change on a per-file basis in the future too.